edx_event_bus_kafka/internal/consumer.py

```diff
@@ -85,45 +96,54 @@ def inner_from_dict(event_data_dict, ctx=None):  # pylint: disable=unused-argume
 
         return DeserializingConsumer(consumer_config)
 
+    def _shut_down(self):
+        """
+        Test utility for shutting down the consumer loop.
+        """
+        self._shut_down_loop = True
+
     def consume_indefinitely(self):
         """
         Consume events from a topic in an infinite loop.
         """
         # This is already checked at the Command level, but it's possible this loop
         # could get called some other way, so check it here too.
         if not KAFKA_CONSUMERS_ENABLED.is_enabled():
-            logger.error("Kafka consumers not enabled")
+            logger.error("Kafka consumers not enabled, exiting.")
             return
 
         try:
             full_topic = get_full_topic(self.topic)
+            run_context = {
+                'full_topic': full_topic,
+                'consumer_group': self.group_id,
+                'expected_signal': self.signal,
+            }
             self.consumer.subscribe([full_topic])
+            logger.info(
+                f"Running consumer for {run_context!r}; assigned {self.consumer.assignment()!r}"
```

Comment:
It's a list of `TopicPartition`: https://docs.confluent.io/platform/current/clients/confluent-kafka-python/html/index.html#confluent_kafka.TopicPartition

We should expect to get a list of one of these. (They print nicely.)

Reply:
For reference, the assignment would print out as something like this: `[TopicPartition{topic=hello,partition=4,offset=-1001,error=None}]`

I think I'll adjust the message to specifically pull out the partition information since the topic is redundant between the `run_context` and the assignment.

Reply:
Hmm no, right, assignments is a list and I don't want to deal with "what if the list isn't of size 1 and/or the topic doesn't match..." -- I've pushed something up, see what you think.

Reply:
From some experimentation and searches, it seems that this assignment can change over time, and will likely not be available before the first poll. So I'll remove that. The `run_context` isn't all that interesting without it, but it could be marginally useful so I guess I'll leave the rest of the line in.